File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
import heapq

class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        max_heap = []

        for num in nums:
            heapq.heappush(max_heap, -num)
        
        kth_largest = 0
        for i in range(k):
            kth_largest = -heapq.heappop(max_heap)

        return kth_largest

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.go
// [1,2,3,4,1,1,,,]

func removeDuplicates(nums []int) int {
	k := 0
	prev := 101

	for i := range nums {
		if nums[i] != prev {
			prev = nums[i]
			nums[k] = nums[i]
			k++
		}
	}
	return k
}
